example/server.py

```diff
@@ -1,11 +1,25 @@
+import pickle
+
 from bottle import Bottle, redirect, run
+import datastore.core
 
 from switchboard import operator, configure
 from switchboard.middleware import SwitchboardMiddleware
 from switchboard.admin import app as switchboard
 
-configure()
 
+# Setup a file-based datastore with pickle serialization.
```

Comment:
What's the reason why these imports cannot be done at the beginning of the file? In the usual 3 import groups?